esp-hal/src/rmt.rs

```diff
@@ -634,6 +645,7 @@ mod impl_for_chip {
     use crate::peripheral::{Peripheral, PeripheralRef};
 
     /// RMT Instance
+    #[allow(missing_docs)]
```

Comment:
Why is this needed? For the inner `pub` channels? Maybe we should just doc them?

Reply:
Yes, it's for the channels inside the struct. 

My thinking is that having "Channel 0","Channel 1" etc. as a comment on every channel is somewhat boring and not very useful 🤔